.changeset/strange-queens-notice.md

```diff
@@ -0,0 +1,5 @@
+---
+"@pnpm/lockfile-types": patch
```

Comment:
Removing a field is usually a breaking change, but since this is a "fix" for a field that's always empty, I opted to keep this as `patch`.